pkg/controller/migmigration/task.go

```diff
@@ -927,7 +927,7 @@ func (t *Task) init() error {
 }
 
 func (t *Task) initPipeline() {
-	for _, phase := range FinalItinerary.Phases {
+	for _, phase := range t.Itinerary.Phases {
```

Comment:
@djwhatle This makes sure only the Steps belonging to the current itinerary are added to pipeline

Reply:
Good catch